server/test/user.spec.js

```diff
@@ -101,6 +102,23 @@ describe('Users Routes', () => {
       expect(res).to.be.a('object');
       expect(res.body).to.have.property('data');
     });
+    it('should not get users profile if authorization token is not valid', async () => {
+      const res = await chai.request(app)
+        .get('/api/v1/users')
+        .set({ Authorization: `Bearer ${invalidToken}` });
+      expect(res).to.have.status(401);
+      expect(res).to.be.a('object');
+      expect(res.body).to.have.property('message').eql('Error in verification. Please try again');
+    });
+    it('should get users profile', async () => {
+      const res = await chai.request(app)
+        .get('/api/v1/users')
+        .set({ Authorization: `Bearer ${userToken}` });
+      expect(res).to.have.status(200);
+      expect(res).to.be.a('object');
+      expect(res.body).to.have.property('message').eql('Successfully retrieved users');
+      expect(res.body.data).to.have.property('rows');
```

Comment:
@geekdanitek Good job. I think we can make this test more comprehensive by checking for one or two users that are seeded in the DB.

Reply:
@cupidcoder I would need clarity on this.

Reply:
@geekdanitek users have been seeded in the DB already. So we can check to ensure that the users being returned are actually from the DB.

Reply:
Ok, will implement this.